Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/worker/WorkerBinder.kt

```diff
@@ -0,0 +1,58 @@
+package de.rki.coronawarnapp.util.worker
+
+import androidx.work.ListenableWorker
+import dagger.Binds
+import dagger.Module
+import dagger.multibindings.IntoMap
+import de.rki.coronawarnapp.nearby.ExposureStateUpdateWorker
+import de.rki.coronawarnapp.worker.BackgroundNoiseOneTimeWorker
+import de.rki.coronawarnapp.worker.BackgroundNoisePeriodicWorker
+import de.rki.coronawarnapp.worker.DiagnosisKeyRetrievalOneTimeWorker
+import de.rki.coronawarnapp.worker.DiagnosisKeyRetrievalPeriodicWorker
+import de.rki.coronawarnapp.worker.DiagnosisTestResultRetrievalPeriodicWorker
+
+@Module
+abstract class WorkerBinder {
```

Comment:
Similar to our viewmodels we add a map of all worker-workerfactory combinations to our DI graph